roles/tackle/templates/persistentvolumeclaim-hub-bucket.yml.j2

```diff
@@ -11,13 +11,13 @@ metadata:
     volume: {{ hub_bucket_volume_name }}
 spec:
   accessModes:
-    - ReadWriteMany
+    - {{ hub_bucket_volume_claim_mode }}
   resources:
     requests:
       storage: {{ hub_bucket_volume_size }}
-{% if rwx_storage_class is defined %}
-{% if rwx_storage_class|length %}
-  storageClassName: {{ rwx_storage_class }}
+{% if cache_storage_class is defined %}
```

Comment:
@jmontleon So use for rwx_storage_class used to encompass the same access mode for maven and bucket , now that this is not the case. I think bucket_storage_class should have its own controlling variable instead of using cache_storage_class.

Existing customers using rwx_storage_class set might see their requested storage class name ignored from now on

Reply:
My understanding is that there is nothing to be gained by having the hub bucket volume created as RWX, which is why it's set to RWO by default now.

As mentioned above I'm aware the accessMode is immutable and there may be existing installs with it set as RWX. That's why we check and override the value with the existing value. New installs should be created as RWO.

Is there a reason we expect a user should want or need to create the hub bucket as an RWX volume? I guess technically they could override it with `hub_bucket_volume_claim_mode`, though my motivation to parameterize this  was more for the sake of allowing the playbook to complete without error when and pv with the old access mode already exists.

Reply:
@fbladilo I think I misunderstood what you were saying. I see we have the cache params in the hub bucket PV. Let me fix.

Reply:
@fbladilo I changed it to hub_bucket_storage_class, which I think is consistent with the other hub bucket parameter names. Let me know if you prefer something else or see any others I missed. Thanks!

Reply:
@fbladilo I wonder if we should do something about your comment regarding rwx_storage_class.... we could do something like if rwx_storage_class defined set_fact cache_storage_class... maybe check that cache_storage_class is not defined in case they become wise to the change and somehow end up with both set... let me know your thoughts, I don't have a strong opinion, so happy to implement whatever you think is best.